PLAYER/SetAllRandomPedsFlee.md

```diff
@@ -8,8 +8,9 @@ ns: PLAYER
 void SET_ALL_RANDOM_PEDS_FLEE(Player player, BOOL toggle);
 ```
 
+Sets whether all random peds will run away from player if they are agitated (threatened) (bool=true), or some peds can stand up for themselves (bool=false).
 
 ## Parameters
-* **player**: 
-* **toggle**: 
+* **player**: PLAYER::PLAYER_ID()
```

Comment:
I think "The player to set this to i.e. `PlayerId()`" would be a better description.